pkg/virt-launcher/virtwrap/cmd-server/server_test.go

```diff
@@ -179,6 +179,38 @@ var _ = Describe("Virt remote commands", func() {
 			Expect(domStats.Name).To(Equal(list[0].Name))
 			Expect(domStats.UUID).To(Equal(list[0].UUID))
 		})
+
+		It("should return full user list", func() {
+			userList := []v1.VirtualMachineInstanceGuestOSUser{
+				v1.VirtualMachineInstanceGuestOSUser{
+					UserName: "testUser",
+				},
+			}
+
+			domainManager.EXPECT().GetUsers().Return(userList, nil)
+
+			fetchedList, err := client.GetUsers()
+			Expect(err).ToNot(HaveOccurred(), "should fetch users without any issue")
+			Expect(fetchedList.Items).To(Equal(userList), "fetched list should be the same")
+		})
+
+		It("should return full filesystem list", func() {
+			fsList := []v1.VirtualMachineInstanceFileSystem{
+				v1.VirtualMachineInstanceFileSystem{
+					DiskName:       "main",
+					MountPoint:     "/",
+					FileSystemType: "EXT4",
+					UsedBytes:      3333,
+					TotalBytes:     9999,
+				},
+			}
+
+			domainManager.EXPECT().GetFilesystems().Return(fsList, nil)
+
+			fetchedList, err := client.GetFilesystems()
+			Expect(err).ToNot(HaveOccurred(), "should fetch filesystems without any issue")
+			Expect(fetchedList.Items).To(Equal(fsList), "fetched list should be the same")
+		})
 	})
 
```

Comment:
Maybe add unit tests for error cases? I.e. domainManager.getFilesystems() fails of json deserialization fails?

Reply:
The tests cover the regression when it happens. I do not see the value of adding the negative test case. Would you agree to skip it?